recipes/nvim/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export LIBTOOL=${BUILD_PREFIX}/bin/libtool
+export LIBTOOLIZE=${BUILD_PREFIX}/bin/libtoolize
+
+make CMAKE_BUILD_TYPE=Release CMAKE_EXTRA_FLAGS="-DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT}"  MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
```suggestion
make CMAKE_BUILD_TYPE=Release CMAKE_EXTRA_FLAGS="-DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT}"  MACOSX_DEPLOYMENT_TARGET=$MACOSX_DEPLOYMENT_TARGET
```

Reply:
> nvim, still vendors a lot of third party packages. Can we use other conda packages for them?

Some no. Here's the rough pictures (conda-forge checked):

- [X] gperf
- [X] luajit
- [X] luarocks
- [X] libuv1-dev
- [?] libluajit-5.1-dev
- [ ] libunibilium-dev
- [X] libmsgpack-dev
- [ ] libtermkey-dev
- [ ] libvterm-dev
- [X] libutf8proc-dev

But some upstream deps might be modified. I need further check.

Reply:
yeah - you may need to add the other deps then

Reply:
Update: no libuv dev package in conda-forge.

Reply:
Update: no luarocks for lua5.1, making all lua related dependencies no go. nvim stick to 5.1 https://github.com/neovim/neovim/wiki/FAQ#why-lua-51-instead-of-lua-53

Reply:
Use `libuv` instead of `libuv-dev`. Package naming varies between different package managers and conda-forge doesn't use `-dev` naming.

This is a good place to try searching: https://conda-forge.org/feedstocks/

Reply:
For luarocks you can open a PR here suggesting a new branch be made for lua 5.1:

https://github.com/conda-forge/luarocks-feedstock

I know it's a bit of a pain to do all of this but having lots of vendored packages causes a mess of weird and wonderful incompatibilities that can be really hard to debug.

Reply:
Thanks for help. I'm still scoping the work to be done. It seems luarocks can be back-compatible: `luarocks --lua-dir=$(brew --prefix)/opt/lua@5.1 install`. Sorry for confusion, I actually found libuv, but i ran into missing include headers. Maybe it's path config problem.

Reply:
Ok, `--lua-dir` is a good idea for `homebrew`, since it has lua@5.1 package coexist with lua and do run time switch in luarocks. I don't know if conda's philosophy is to avoid such case. I think luarocks should be subpackage in lua instead of standalone generally. But now let's see if they would accept PR this one: https://github.com/conda-forge/luarocks-feedstock/pull/21